enterprise/api.py

```diff
@@ -1,27 +0,0 @@
-# -*- coding: utf-8 -*-
-"""
-Helper functions for enterprise app.
-"""
-from __future__ import absolute_import, unicode_literals
-
-from enterprise.models import EnterpriseCustomerBrandingConfiguration
-
-
-def get_enterprise_branding_info_by_provider_id(provider_id=None):  # pylint: disable=invalid-name
```

Comment:
@saleem-latif we need to log a ticket or a new PR to update edx-platform code according to these changes. 
ref: https://github.com/edx/edx-platform/blob/master/common/djangoapps/util/enterprise_helpers.py#L10